package.json

```diff
@@ -107,6 +108,7 @@
     "@houdiniproject/react-i18n-currency-input": "^2.0.0-pre3",
     "@material-ui/core": "^4.10.2",
     "@material-ui/icons": "^4.9.1",
+    "@material-ui/lab": "^4.0.0-alpha.56",
```

Comment:
Why was this added?

Reply:
Any update here?

Reply:
We are not sure but we removed it

Reply:
We decided to use the lab feature so I'll resolve this.